core/src/main/scala/kafka/server/KafkaConfig.scala

```diff
@@ -835,6 +836,7 @@ object KafkaConfig {
       .define(CreateTopicPolicyClassNameProp, CLASS, null, LOW, CreateTopicPolicyClassNameDoc)
       .define(AlterConfigPolicyClassNameProp, CLASS, null, LOW, AlterConfigPolicyClassNameDoc)
       .define(LogMessageDownConversionEnableProp, BOOLEAN, LogConfig.DEFAULT_MESSAGE_DOWNCONVERSION_ENABLE, LOW, LogMessageDownConversionEnableDoc)
+      .defineInternal(LogInitialTaskDelayMsProp, LONG, LogConfig.DEFAULT_INITIAL_TASK_DELAY_MS, LOW)
```

Comment:
Could we add a doc at the last parameter for other developers know what this config is doing for? 
Ex: 
The initial task delay in millisecond when initializing tasks in LogManager. This should be used for testing only.

Reply:
Also, we can set `atLeast(0)` in the defineInternal method. So that we don't need additional validator below.